contrib_bots/lib/github_detail.md

```diff
@@ -0,0 +1,27 @@
+# GitHub detail bot
+This bot links and details issues and pull requests.
+To use it username mention the bot then type an id:
```

Comment:
I think it's clear enough as it is. In addition the username mention isn't static, it's whatever the bots current username is.